graphql_compiler/compiler/emit_sql.py

```diff
@@ -103,6 +105,24 @@ def _find_folded_outputs(ir):
     return folded_outputs
 
 
+# 3-tuple describing the join information for each traversal in a fold.
+#
+# Contains join descriptor naming the columns used in the join predicate,
+# the source/from table, and the destination/to table
+SQLFoldTraversalDescriptor = namedtuple('SQLFoldJoinInfo', (
+    # SQLAlchemySchemaInfo join descriptor giving columns used to join from_vertex/to_vertex
+    'join_descriptor',
+
+    # SQLAlchemy table corresponding to corresponding to the outside vertex of the traversal,
+    # appears on the left side of the join.
+    'from_vertex',
```

Comment:
It's not intuitive that `from_vertex` is a table. Call it `from_table`?

Reply:
This is the terminology that we use for `DirectEdgeDescriptor` objects. I think that we should have a consistent naming convention. If we make this change here then we should also probably do it in schema generation.

Reply:
Sorry I did not notice this was referring to SQLAlchemy Table object. I thought that it was referring to the vertex name of the SQLAlchemy Table. `from_table` seems like a good name.